boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -195,3 +195,27 @@ ipc0: &cpuapp_cpurad_ipc {
 	pinctrl-names = "default", "sleep";
 	hw-flow-control;
 };
+
+&exmif {
+	cs-gpios = <&gpio6 3 GPIO_ACTIVE_LOW>;
+	pinctrl-0 = <&exmif_default>;
+	pinctrl-names = "default";
+	mx25uw63: mx25uw6345g@0 {
+		compatible = "jedec,spi-nor";
+		status = "disabled";
```

Comment:
why is it disabled at board level?

Reply:
I can enable it by default, if that's the recommendation. On nRF54H20 DK the pins used by EXMIF/external flash are not connected to anything else.

Reply:
at board level things are usually enabled by default.

Reply:
Enabled EXMIF by default, but flash node is kept disabled, like it was done here: https://github.com/zephyrproject-rtos/zephyr/blame/main/boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp.dts#L135

Reply:
why?

Reply:
I wanted to avoid having `CONFIG_SPI_NOR` enabled in all samples by default. Setting status okay would result in `CONFIG_SPI_NOR` being enabled by default when `CONFIG_FLASH=y`. On current nRF54H20 DKs, powering up the flash chip requires some additional steps from the user.